src/dotnet/UI.Blazor.App/Pages/Landing/landing.css

```diff
@@ -237,19 +238,23 @@ body.app-maui .landing-header .download-btn {
 }
 
 /* Page 1 styles */
-.landing .page-1 {
+.landing .page-1,
+.landing .page-links {
     @apply bg-[#0436A3];
 }
-.landing .page-1 .page-1-wrapper {
+.landing .page-1 .page-1-wrapper,
+.landing .page-links .page-links-wrapper {
     @apply h-full;
     @apply pt-16;
     @apply bg-contain bg-no-repeat;
     @apply bg-[url('/dist/images/ellipses.svg')];
 }
-body.narrow .landing .page-1 {
+body.narrow .landing .page-1,
+body.narrow .landing .page-links {
```

Comment:
I think we can just use 'page-1' class on page-links page. Then you don't need to extend your css classes.